(unknown location): internal error, Invalid_argument("TODO: Con in dot operator")

Last environment:

